File: src/prog_langs/php/foreach_dogs.php
<?php
	$x = ['test','hoge','piyo','huga'];// 元となる配列
	$dogs = [];// 検索データを格納する空の配列

	foreach ($x as $key => $value) {
		$dogs[] = $value.'🐶';// $dogsに犬をつける
	}

	foreach ($dogs as $key => $value) {// 作成した$dogs配列を回す
		echo $value."\n";// すべてecho
	}
	// 実行結果
	// test🐶
	// hoge🐶
	// piyo🐶
	// huga🐶
?>

File: src/prog_langs/python/foreach_dogs.py
dogs  = []# 空の配列を作成
array = ['test', 'hoge', 'piyo', 'huga']

for val in array:
    # eが含まれる場合のみ空の配列にappendと🐶
    dogs.append(val+"🐶")

for v in dogs:# 全て回す
    print(v)

# 実行結果
# test🐶
# hoge🐶
# piyo🐶
# huga🐶